### PR TITLE
feat: add an .excalidraw prefix to the extension of image export with embedded sceneData

### PR DESCRIPTION
**Pull request by pomdtr**
This will have several benefits:
- the user can now if an image embed a scene by just looking at the filename
- it also allows better integration with the vscode extension (file exported from excalidraw can be directly opened using the vscode extension)

### CODE REVIEW

File: src/constants.ts [hunk 1]
```diff
@@ -94,7 +94,9 @@ export const MIME_TYPES = {
   excalidrawlib: "application/vnd.excalidrawlib+json",
   json: "application/json",
   svg: "image/svg+xml",
+  "excalidraw.svg": "image/svg+xml",
   png: "image/png",
+  "excalidraw.png": "image/png",
```
Comment (dwelle) [inline src/constants.ts:99]: this is unused and probably will always be, so let's remove

Comment (dwelle) [reply src/constants.ts:99]: wait, it's used for types somewhere

Comment (pomdtr) [reply src/constants.ts:99]: Yep, the compilation did break until I changed it

Comment (dwelle) [reply src/constants.ts:99]: ok, let's leave it then

**Review by dwelle — APPROVED** [review]

**Pull request merged**